### DISCUSSION TITLE
Changelog in releases

### DISCUSSION THREAD

**Discussion by John-H-Smith** [kromitgmbh/titra#167] [Ideas]
Hi, It would be highly appreciated if changelog notes would be included into the release details. Right now, one can not know about any changes. 
Best for that I would suggest conventional commits: [https://www.conventionalcommits.org/de/v1.0.0/](https://www.conventionalcommits.org/de/v1.0.0/)

**Comment by faburem** [comment]
I am not sure if I can follow, all releases are tagged with all the changes they include - what exactly are you missing?
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/11456790/226728861-d843d1ba-7cd8-46f8-80dd-6804c1f44b3e.png">

**Comment by John-H-Smith** [comment]
OOf, maybe I should have checked the releases on the desktop. On mobile, there's no description in the releases:
![Screenshot_20230322_070417_GitHub](https://user-images.githubusercontent.com/18233867/226816528-84793b5b-60dc-458e-816b-50d31d0e0b63.jpg)
As this seems to be a bug with GitHub, please excuse my discussion.
